Use 'unselectable' property in LabelState

fabrixxm opened the pull request:
this fixes #25

alexislozano commented on src/widgets/label.rs:
```diff
@@ -158,8 +158,8 @@ impl Label {
     }
 
     /// Set the selectable flag to false
```
unselectable flag

alexislozano commented on src/widgets/label.rs:
```diff
@@ -175,16 +175,16 @@ impl Widget for Label {
         };
-        let selectable = if self.state.selectable() {
-            "selectable"
+        let selectable_class = if self.state.unselectable() {
```
"unselectable" and "selectable" class ? Just want confirmation ;)

fabrixxm replied:
it's a not nice name for the var, it should be something like `selectable_status_css_class`, but it's too long for my taste :)
I don't like to use the name `unselectable` because the vaule should be `""` or `"selectable"`, as the css class in app.scss is `.selectable`, to override the default role.

alexislozano replied:
i meant you have something like
```rust
if ... {
    "selectable"
} else {
    "unselectable"
}

alexislozano replied:
i am talking about the css class :)

alexislozano replied:
i meant you have something like
```rust
if ... { "selectable" } else { "unselectable" }
you should either have
```rust
if ... { "selectable" } else{ "" } (or if and else reversed)
```
or 
```rust
if ... { "unselectable" } else { "" } (or if and else reversed)
```

alexislozano commented on src/widgets/label.rs:
```diff
@@ -28,8 +28,8 @@ impl LabelState {
     }
 
     /// Get the selectable flag
```
unselectable here too

alexislozano replied:
comment

alexislozano commented on src/widgets/label.rs:
```diff
@@ -43,8 +43,8 @@ impl LabelState {
     }
 
     /// Set the selectable flag
-    pub fn set_selectable(&mut self, selectable: bool) {
```
unselectable here too

alexislozano replied:
talking about comment

fabrixxm commented:
well this PR shows us why one should not open a PR while on train back home after a long Monday...
I'll review all this mess tomorrow... sorry.

alexislozano commented:
No problem, have a great evening ;)

fabrixxm commented:
I've fixed the comments, I added css class `unselectable` class along `selectable`, I've leaved both class in the `if` in `eval()` : i find it more clear this way instead of using a reverse logic.
